recipe/meta.yaml

```diff
@@ -9,12 +9,12 @@ source:
   sha256: bc10c33bfdcaa4e749b779f62f60d6e12f8215c46a292d05e486b869ae306619
 
 build:
-  number: 0
-  noarch: python
-  script: python -m pip install --no-deps --ignore-installed .
+  number: 1
+  script: python setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
I believe you don't need to move it back to the old way. `pip` should build the extension correctly here.
Can you revert this change?

Reply:
OK, switched back to the original form.